azure-iot-device/tests/iothub/transport/test_pipeline_events_iothub.py

```diff
@@ -0,0 +1,29 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+from azure.iot.device.iothub.transport import pipeline_events_iothub
+
+fake_message = "__fake_messsage__"
+fake_input_name = "__fake_input_name__"
+
+
+@pytest.mark.describe("C2DMessage object")
```

Comment:
>C2DMessage object [](start = 23, length = 17)

This is probably going to be confusing since the event is called C2DMessage, which makes it sound like a more specific version of the Message class.

Perhaps events should have "Event" in the title, i.e. C2DMessageEvent to avoid confusion?